python/tests/res/fm/test_fm_config.py

```diff
@@ -0,0 +1,39 @@
+import os
+import unittest
+from tests import ResTest
+
+class TestFMValidity(ResTest):
+
+    def setUp(self):
+        pass
+
+    def _extract_executable(self,filename):
+        executable_list = []
+        with open(filename,'r') as f:
+            for line in f.readlines():
+                l = [a.strip() for a in line.split(' ') if a not in ['', ' ', '  ']]
+                if l[0]=='EXECUTABLE':
+                    executable_list.append(l[1])
+        return executable_list
+
+    def _file_exist_and_is_executable(self,file_path):
+        return os.path.isfile(file_path) and os.access(file_path, os.X_OK)
+
+    def test_validate_scripts(self):
+        fm_path = ['share/ert/forward-models/res',
```

Comment:
I would prefer if this was a listdir of the forward model directory. There are definitely coming more folders here in the future and at some point someone will forget to update this test..

Reply:
Ok, sounds legit